Vagrantfile

```diff
@@ -244,6 +244,18 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
                     end
                     formula_path
                 }
+                # write the minion file
+                minion_cfg = YAML.load_file("scripts/salt/minion.template")
+                more_base_paths = formula_paths.map{|p| "/vagrant/" + p + "/"}
+                minion_cfg['file_roots']['base'].insert(1, *more_base_paths)
```

Comment:
yes, it preserves the ordering in the template with the custom salt directory first for any overrides and the project-local /srv/salt at the end. the funny `.insert(1, *more_base_paths)` is something I've not seen done in Python. It does `[1 2 3].insert(1, *[a b c]) => [1 a b c 2 3]`